packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -76,4 +81,40 @@ export default class Proxy {
     const { properties } = await this._db.update(proxy)
     return properties
   }
+
+  // specification: https://github.com/vatesfr/xen-orchestra/blob/xo-proxy/@xen-orchestra/proxy/docs/api.md
```

Comment:
IMHO we should remove this comment because this branch will be deleted when merged into master.